README.md

```diff
@@ -1,49 +1,34 @@
 # macOS Cookbook
 
-[![Build Status](https://dev.azure.com/office/OE/_apis/build/status/Microsoft.macos-cookbook?branchName=master)](https://dev.azure.com/office/OE/_build/latest?definitionId=5072&branchName=master)
+[![Build Status](https://dev.azure.com/office/OE/_apis/build/status/microsoft.macos-cookbook?repoName=microsoft%2Fmacos-cookbook&branchName=master)](https://dev.azure.com/office/OE/_build/latest?definitionId=17314&repoName=microsoft%2Fmacos-cookbook&branchName=master)
 
 Chef resources and recipes for managing and provisioning macOS.
 
-- [Chef Requirements](#chef-requirements)
-- [Supported OS Versions](#supported-os-versions)
-- [Attributes](#attributes)
-- [Recipes](#recipes)
-- [Data Bags](#data-bags)
-- [Resources](#resources)
+- [Chef Requirements](#officially-supported-chef-versions)
+- [macOS Requirements](#officially-supported-os-versions)
+- [Included Resources](#resources)
 
 ## Officially Supported Chef Versions
 
 - Chef 16
 - Chef 17
 
-## Supported OS Versions
+## Officially Supported OS Versions
 
-- macOS High Sierra 10.13
-- macOS Mojave 10.14
-- macOS Catalina 10.15
-- macOS Big Sur 11.0
-
-## Attributes
-
-### Admin User and Password
-
-```ruby
-node['macos']['admin_user'] = 'vagrant'
-node['macos']['admin_password'] = 'vagrant'
-```
-
-Each of these attributes defaults to vagrant since our resources are developed
-with the Vagrant paradigm. In other words, the user and password declared here
-should be an admin user with passwordless super-user rights.
+- macOS 10.14 Mojave
+- macOS 10.15 Catalina
+- macOS 11 Big Sur
+- macos 12 Monterey
 
 ## Resources
+**The plist resource is deprecated and will be removed in the next major release of macos-cookbook. Please use the plist resource included with Chef Client >=16.**
```

Comment:
The macos-cookbook's implementation of `plist` supports the ability to specify arrays, but the Chef Client implementation does not:
> value
> Ruby Type: true, false, String, Integer, Float, Hash

https://docs.chef.io/resources/plist/

Reply:
That's odd - it should since it was ported directly from this cookbook. It seems like functionality is there https://github.com/chef/chef/blob/a7b347ceb49f66ee5797ae578fbbcc4e45983aee/lib/chef/resource/plist.rb#L164, but is not listed in the value types.

Reply:
I'm not sure the macos implementation explicitly supports arrays either: https://github.com/microsoft/macos-cookbook/blob/103fab4446c74dde8cf8412e936e1584f59d09f5/resources/plist.rb#L7

Reply:
Your mention in the other comment thread about `defaults` resource made me realize that I may be getting the two resources confused. Certainly the issue I had previously filed about support for arrays was for the `defaults` resource, so that's probably it.
#239